src/components/FreeTokensDialog.vue

```diff
@@ -55,19 +56,20 @@ import { websiteUriToOnion } from '@/lib/uri'
 
 export default {
   props: {
-    value: {
+    modelValue: {
       type: Boolean,
       required: true
     }
   },
+  emits: ['update:modelValue'],
   computed: {
     className: () => 'free-tokens-dialog',
```

Comment:
`className` is used inside the template

Reply:
but why should the const string be computed?

Reply:
You cant use constants inside the <template/>. Now in Vue 3 it's possible, but the component must be refactored to Composition API first. This will be done shortly, not in current PR.

Reply:
got it